IronFrame/Utilities/FirewallManager.cs

```diff
@@ -43,6 +50,87 @@ public void OpenPort(int port, string name)
             firewallPolicy.Rules.Add(firewallRule);
         }
 
+        internal static string GetFormattedLocalUserSid(string windowsUsername)
```

Comment:
Does this need to be internal?  Are you calling it from somewhere else?  Seems like just a private helper function.